.gitignore

```diff
@@ -14,3 +14,4 @@ test/out-fixtures/*
 test/output/
 test/watch-*.txt
 gulp.1
+package-lock.json
```

Comment:
Makes sense but does npm pick up `.npmrc` files from the repo?  Might be better to include an npmrc that stops the package-lock creation.  Also, belongs in a separate commit.

Reply:
I got it. I'll commit .npmrc separately.